projects/nae-example-app/src/netgrif-theme.scss

```diff
@@ -26,30 +24,30 @@ $netgrif-blue: (
     A400: #ff9100,
     A700: #ff6d00,
     contrast: (
-        50: $dark-primary-text,
-        100: $dark-primary-text,
-        200: $dark-primary-text,
-        300: $dark-primary-text,
-        400: $dark-primary-text,
-        500: $light-primary-text,
-        600: $light-primary-text,
-        700: $light-primary-text,
-        800: $light-primary-text,
-        900: $light-primary-text,
-        A100: $dark-primary-text,
-        A200: $dark-primary-text,
-        A400: $dark-primary-text,
-        A700: $dark-primary-text,
+        50: rgba(black, 0.87),
```

Comment:
as i looked to documentation of material angular, they also show there static colors instead of $dark-primary-text and $light-primary-text, and I think the schematic from material 12 to 13 changed it to static colors, so I think it should be like that